puma.sh

```diff
@@ -1,89 +0,0 @@
-#!/usr/bin/env bash
```

Comment:
Once this is merged, I will then write upstart scripts to manage nginx, puma and delayed job so that they will be automatically started when the server reboots